service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EhrExtractMapperTest.java

```diff
@@ -86,7 +86,8 @@ public void setUp() throws IOException {
                 messageContext,
                 new StructuredObservationValueMapper(),
                 new PertinentInformationObservationValueMapper(),
-                codeableConceptCdMapper
+                codeableConceptCdMapper,
+                new ParticipantMapper()
```

Comment:
Do we need to update any of the json/xml extracts to include a performer/participant?